java/src/test/java/com/saucelabs/simplesauce/unit/DataCenterTest.java

```diff
@@ -0,0 +1,17 @@
+package com.saucelabs.simplesauce.unit;
+
+import com.saucelabs.simplesauce.DataCenter;
+import org.hamcrest.text.IsEqualIgnoringCase;
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class DataCenterTest
+{
+    @Test
+    public void usWestDataCenterUrl_isCorrect()
+    {
+        assertThat(DataCenter.USWest,
+                IsEqualIgnoringCase.equalToIgnoringCase("https://ondemand.saucelabs.com/wd/hub"));
```

Comment:
We shouldn't write unit tests just to verify strings like this... it's brittle, and doesn't really test anything important. Don't delete it, however--when we resolve issue-17, this test will start to become interesting

Reply:
@mmerrell that's interesting. I'm curious as to why. The reason I've always tested this is for example: Let's pretend that someone decides to change the DataCenter.USWest url to something else, like Titus had "blah.blah:443". With pretty much all string concatenation, shit can always go wrong and you don't know how a new string will work where it's used. So usually, I validate my requirement strings so that if someone decides to change it, I have tests screaming that you broke something.
Btw, not disagreeing with you 👍  Simply trying to understand the logic as I'm sure you have more unit testing experience than I do. Let me know your thoughts